### PR TITLE
Remove defaults from the config file

### PR DESCRIPTION
**Pull request by jaydenmilne**

### CODE REVIEW

**Comment by jaydenmilne** [comment]
See BYU-PCCL/holodeck#206

File: DefaultWorlds/AndroidPlayground-MaxDistance.json [hunk 1]
```diff
@@ -7,75 +7,31 @@
             "agent_type": "AndroidAgent",
             "sensors": [
                 {
-                    "sensor_name": "RGBCamera",
-                    "sensor_type": "RGBCamera",
-                    "location": [0.0, 0.0, 0.0],
-                    "rotation": [0.0, 0.0, 0.0],
-                    "socket": "CameraSocket",
```
Comment (MaxRobinsonTheGreat) [inline DefaultWorlds/AndroidPlayground-MaxDistance.json:14]: keep CameraSocket

File: DefaultWorlds/CyberPunkCity-FollowSight.json [hunk 2]
```diff
@@ -7,59 +7,25 @@
             "agent_type": "UavAgent",
             "sensors": [
                 {
-                    "sensor_name": "RGBCamera",
-                    "sensor_type": "RGBCamera",
-                    "location": [0.0, 0.0, 0.0],
-                    "rotation": [0.0, 0.0, 0.0],
-                    "socket": "",
```
Comment (MaxRobinsonTheGreat) [inline DefaultWorlds/CyberPunkCity-FollowSight.json:14]: Add CameraSocket

File: DefaultWorlds/EuropeanForest-MaxDistance.json [hunk 3]
```diff
@@ -7,59 +7,25 @@
             "agent_type": "UavAgent",
             "sensors": [
                 {
-                    "sensor_name": "RGBCamera",
-                    "sensor_type": "RGBCamera",
-                    "location": [0.0, 0.0, 0.0],
-                    "rotation": [0.0, 0.0, 0.0],
-                    "socket": "",
```
Comment (MaxRobinsonTheGreat) [inline DefaultWorlds/EuropeanForest-MaxDistance.json:14]: CameraSocket

**Review by MaxRobinsonTheGreat — CHANGES_REQUESTED** [review]
Need to add camerasocket

File: DefaultWorlds/EuropeanForest-MaxDistance.json (hunk 3 above)
Comment (MaxRobinsonTheGreat) [reply DefaultWorlds/EuropeanForest-MaxDistance.json:14]: I'm just gonna approve and add the changes in a different branch

**Review by MaxRobinsonTheGreat — APPROVED** [review]

**Pull request merged**